Modules/Indexer/DialsIndexer.py

```diff
@@ -487,7 +485,8 @@ def _index(self):
                 elif nref_3d is not None:
                     indexer = indexer_fft3d
                 else:
-                    raise RuntimeError(e)
+                    # Flake8 doesn't recognise this as `e` from above.
```

Comment:
Aaaaaah, this is different behaviour in Python 3, the value of `e` isn't retained from the `except` statement above.